board/fischertechnik/TXT/rootfs/etc/init.d/S80touchscreen-calibration

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+start() {
+  if [ -f /etc/reset-touchscreen-calibration ]; then
+    cp /media/sdcard/root/etc/pointercal /etc/pointercal
+    rm /etc/reset-touchscreen-calibration
```

Comment:
This will restore the calibration data to the values delivered with the firmware and not to the previous values. Ist this intended?

Reply:
Yes. The values from fischertechnik are good enough to work with and the old values from the user might be even worse